services/jira/jira-base.js

```diff
@@ -11,10 +11,19 @@ module.exports = class JiraBase extends BaseJsonService {
   async fetch({ url, qs, schema, errorMessages }) {
     const options = { qs }
 
-    if (serverSecrets && serverSecrets.jira_username) {
+    if (
+      serverSecrets &&
+      (serverSecrets.jira_user || serverSecrets.jira_username)
+    ) {
+      /*
+      for legacy reasons we still allow
+      jira_username, jira_password
+      but prefer
+      jira_user, jira_pass
```

Comment:
Is this change to bring these more in line with the others?

Reply:
Yep - as noted in the commit message for 523ba3f :


> All the other services use servicename_user, servicename_pass

> This switches JIRA to use that convention by preference but supports _username and _password for legacy users.


While I was looking at all of them in one place, it seemed like a good idea to bring JIRA into line, but I'm not married to that decision. If we think this is more hassle than its worth, I can back out that change and we can just document that the keys are `bintray_user`, `jenkins_user`, `nexus_user` but `jira_username`.